apps/service_define/lib/define/data_definition.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Define.DataDefinition do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "data_definition" do
+    field(:dataset_id, :string)
+    field(:subset_id, :string, default: "default")
+    # {:map, inner_type}
```

Comment:
This guy is a holdover from a while ago, I believe.
```suggestion
```

Reply:
While file removed